### DISCUSSION TITLE
Should we remove console logging?

### DISCUSSION THREAD

**Discussion by oeyoews** [mermaid-js/mermaid#5771] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/0078059e-ddb4-4df8-8a93-f3e17965ae48)
Some diagrams print some logs to the browser console, should we remove them? @sidharthv96

**Comment by sidharthv96** [comment, marked as answer]
Those are debug logs, useful while developing. There are options to set log levels, which would help when people are integrating mermaid to their site and don't want logs. 

In live editor, it's okay to have logs, as it could help with debugging.
